Branch-SDK/Branch-SDK/BNCServerRequestQueue.m

```diff
@@ -47,7 +45,7 @@ - (void)enqueue:(BNCServerRequest *)request {
 - (void)insert:(BNCServerRequest *)request at:(unsigned int)index {
     @synchronized(self.queue) {
         if (index > self.queue.count) {
-            [BNCPreferenceHelper log:FILE_NAME line:LINE_NUM message:@"Invalid queue operation: index out of bound!"];
+            [[BNCPreferenceHelper preferenceHelper] log:FILE_NAME line:LINE_NUM message:@"Invalid queue operation: index out of bound!"];
```

Comment:
this is the singleton syntax i mentioned above. again, not a big deal, but better have some consistency.